PPAK_CVS/Sources/Scenes/Home/HomeVC.swift

```diff
@@ -83,59 +81,60 @@ final class HomeViewController: BaseViewController {
 
   // MARK: - Event
 
-  private func bindHeader() {
+  func bind(viewModel: HomeViewModel) {}
 
-    // -----------------------------------
-    // ---------------INPUT---------------
-    // -----------------------------------
+  private func bindHeader() {
+    guard let viewModel = viewModel else { return }
 
-    // 콜렉션뷰 헤더 최초 생성
-    viewModel.input.ignoreReusableView.accept(true)
+    // MARK: - Input
 
     // 현재 편의점 로고 버튼 클릭
     header.cvsButton.rx.tap
-      .bind(to: viewModel.input.currentCVSButtonTapped)
+      .map { HomeViewModel.Action.currentCvsButtonTapped }
+      .bind(to: viewModel.action)
       .disposed(by: disposeBag)
 
     // 필터 버튼 클릭
     header.filterButton.rx.tap
-      .bind(to: viewModel.input.filterButtonTapped)
+      .map { HomeViewModel.Action.filterButtonTapped }
+      .bind(to: viewModel.action)
       .disposed(by: disposeBag)
 
     // 편의점 드롭다운 리스트 버튼 클릭
     cvsDropdownView.buttonEventSubject
-      .bind(to: viewModel.input.dropdownCVSButtonTapped)
+      .map { HomeViewModel.Action.cvsButtonTappedInDropdown($0) }
+      .bind(to: viewModel.action)
       .disposed(by: disposeBag)
 
     // 필터 드롭다운 리스트 버튼 클릭
     filterDropdownView.buttonEventSubject
-      .bind(to: viewModel.input.dropdownFilterButtonTapped)
+      .map { HomeViewModel.Action.filterButtonTappedInDropdown($0) }
+      .bind(to: viewModel.action)
       .disposed(by: disposeBag)
 
     // 페이지 컨트롤 인덱스 감지
     header.pageControl.pageIndexSubject
       .skip(1)
       .distinctUntilChanged()
-      .bind(to: viewModel.input.pageControlIndexEvent)
+      .map { HomeViewModel.Action.pageControlIndexEvent($0) }
+      .bind(to: viewModel.action)
       .disposed(by: disposeBag)
 
     // 빈공간 터치 감지
     view.rx.tapGesture(configuration: { _, delegate in
       delegate.simultaneousRecognitionPolicy = .never
     })
-      .map { _ in Void() }
-      .bind(to: viewModel.input.touchBackgroundEvent)
+      .map { _ in return HomeViewModel.Action.backgroundTapped }
```

Comment:
컨벤션을 통일하기 위해 `return`은 생략해도 좋을 거 같아요.
```suggestion
      .map { _ in HomeViewModel.Action.backgroundTapped }
```

Reply:
수정완료!